recipes/srt/all/conanfile.py

```diff
@@ -37,13 +37,7 @@ def _has_stdcxx_sync(self):
 
     @property
     def _has_posix_threads(self):
-        return not (
-            self.settings.os == "Windows"
-            and (
-                is_msvc(self)
-                or (self.settings.compiler == "gcc" and self.settings.compiler.get_safe("threads") == "win32")
-            )
-        )
+        return not (is_msvc(self) or self.settings.compiler.get_safe("threads") == "win32")
```

Comment:
No it's broken, keep previous condition please.

Reply:
Are you sure. Here's the reasoning for the simplifications I made:
1. The original expression with `is_msvc(self)` substituted:
  ```python
not (self.settings.os == "Windows" and (is_msvc(self) or (self.settings.compiler == "gcc" and self.settings.compiler.get_safe("threads") == "win32")))
```
2. The GCC check can be dropped since we are clearly not using Posix threads if `compiler.threads == "win32"`.
  ```python
not (self.settings.os == "Windows" and (is_msvc(self) or self.settings.compiler.get_safe("threads") == "win32"))
```
3. The `os == "Windows"` check is redundant, since both `is_msvc(self)` and `compiler.threads == "win32"` imply it.
  ```python
not (is_msvc(self) or self.settings.compiler.get_safe("threads") == "win32")
```

Reply:
Indeed, my bad 😃